docker/ci/test_webapp.sh

```diff
@@ -17,7 +17,7 @@ check_service_status "$container" /etc/service/nginx
 check_service_status "$container" /etc/service/uwsgi
 
 # Wait for UWSGI to be ready to accept connections.
-timeout 120s grep -q 'WSGI app .* ready in [0-9]* seconds' <(docker logs "$container" -f 2>&1)
+timeout 240s grep -q 'WSGI app .* ready in [0-9]* seconds' <(docker logs "$container" -f 2>&1)
```

Comment:
Also, wow! Is it really taking four minutes to spin up?

Reply:
No, looking at the logs where it failed, it seems like it was taking about 130s, just beyond the previous timeout. But I figured I'd give it a bit of a safety factor and just double what we had before.